warehouse/accounts/services.py

```diff
@@ -55,6 +61,11 @@ def __init__(self, session, ratelimiters=None):
             argon2__parallelism=6,
             argon2__time_cost=6,
         )
+        self.serializer = URLSafeTimedSerializer(
+            settings["password_reset.secret"],
+            salt="password-reset",
+        )
+        self.token_max_age = 21600  # 6 hours
```

Comment:
This is a _little_ tight which I'm OK with but perhaps should be configurable.

Also may be worth letting users know in flashed message as well as the contents itself that they only got 21600 seconds to get the job done.

Reply:
so, as I understand it, a user has 6 hours between requesting a reset and completing it?  If so, i agree, we should communicate it in the UI.  Also, in the reset email.  "This link will expire in 6 hours".

Reply:
that understanding is correct @nlhkabu